test/functional/apm.test.js

```diff
@@ -9,6 +9,7 @@ const ignoreNsNotFound = shared.ignoreNsNotFound;
 const loadSpecTests = require('../spec').loadSpecTests;
```

Comment:
Maybe we rename this to `command_monitoring.test.js` to match the directory name change? Or would you rather do that as part of the greater test cleanup? I'm fine either way.

Reply:
same thoughts here as in my reply on the other PR, but I don't mind if you think it's worth doing now

Reply:
Same as well. LGTM.